CrossPlatformDesktopProject/CrossPlatformDesktopProject/WorldItem/HeartIdle.cs

```diff
@@ -0,0 +1,31 @@
+﻿using CrossPlatformDesktopProject.WorldItem;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System.Collections.Generic;
+
+namespace CrossPlatformDesktopProject.WorldItem
+{
+    class HeartIdle : IWorldItem
+    {
+        private Item item;
```

Comment:
It doesn't look like `item` is used for anything, but in the future it may be that `item` has a specific type (in this case, it could be `heartItem`) which would affect its behavior (such as picking it up increases health)

Reply:
Oh my bad, no it doesn't do anything so I removed it for now. But what Brent says makes sense.